content/hardware/05.nicla/boards/nicla-sense-me/tutorials/nicla-battery-app-note/nicla-battery-app-note.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Understanding Battery Capabilities Of The Nicla Sense ME'
+difficulty: intermediate
+description: 'Make use of the BQ2 your Nicla application.'
+tags:
+  - Battery
+  - Nicla
+  - BQ25120
+
+author: 'Ali Jahangiri'
+hardware:
+  - hardware/01.mkr/05.boards/nicla-sense-me
+software:
+  - Arduino IDE 2.x
+---
+
+## Introduction
+
+We have developed the Nicla Sense ME for low-power operation, and it really shines when you can use it independantly of a USB cable, especially for wireless sensing applications. The Nicla Sense ME includes a low-profile connector that will allow you to connect a LiPo battery. In this application note, we will take a closer look at the battery capabilities of the Nicla boards and how you can use it effectively.
+
+### Goals
+The goals of this project are to:
+- Learn how to identify connector polarity
+- Understand the features of the BQ25120 IC
+- Set charging current on the Nicla Sense ME
+- 
+
+### Hardware & Software Needed
+- [Arduino Nicla Sense ME](https://store.arduino.cc/products/nicla-sense-me)
+- Multimeter
+- IDE 2.x
+- LiPo battery with 3-pin 1.2mm ACH Battery Male Connector
+
+### NTC Battery protection
+You can learn about Li-Ion batteries in the [MKR Battery Application Note](/tutorials/mkr-wifi-1010/mkr-battery-app-note). Briefly, the Nicla Sense ME uses a single cell Li-Ion battery. However, this must include a NTC cable, and also comes with a different connector to save space.
```

Comment:
Does the Nicla work with batteries that do not have an NTC? I ask this because it is more common to find batteries without the NTC. With the Nicla, the battery connector is quite difficult; the NTC adds complexity. I would use a battery without an NTC and mention it for more secure applications where the temperature is crucial for the battery's lifespan.

Reply:
According to HW, my understanding is that the NTC is required for correct and safe operation. I haven't got one yet to test it (stuck in customs).
```suggestion
You can learn about Li-Ion batteries in the [MKR Battery Application Note](/tutorials/mkr-wifi-1010/mkr-battery-app-note). Briefly, the Nicla Sense ME uses a single cell Li-Ion battery. However, this must include a NTC cable, and also comes with a different connector to save space.
*** While a battery without a NTC sensor will work, it should be connected for enhanced safety. ***
```